### DISCUSSION TITLE
New Version 3.7.0 --> 4.0

### DISCUSSION THREAD

**Discussion by dzpex** [jamulussoftware/jamulus#1167] [General]
Considering the large number of updates and improvements to the UI I think it is interesting to consider this version as a major update. In this vision new version will be 4.0. What do you think?

**Comment by hoffie** [comment]
Thanks for this suggestion! My personal impression is that there have been lots of discussions and ideas about future UI improvements during the 3.7.0 development window. The list of actual UI-related changes for 3.7.0 does not seem to be that long though. Are you referring to something specific?

**Comment by dzpex** [comment]
Not in particular...I think for example at the new background and the new feature in development for the UI, or the new AnygenreXX server division....and the other function implemented....i think this are the start-up of a new Jamulus Experience....

**Reply by hoffie** [reply]
I agree that the UI redesign plans may be a rather big thing. Most of it is still in discussion/mock up stage, as far as I remember. So this would rather be part of some future release.

Personally, I'd consider the new Directory servers and the background fix rather small (but necessary and positive) changes which would not immediately make me reach for large version jumps. :)

So, for me, I'd say this release should be 3.7.0 and a larger bump may be adequate in future? :)

Glad to hear that you like all that and thanks for sharing your impressions! :)

**Reply by dzpex** [reply]
it is a privilege for me contributing in a positive way...!
